.travis.yml

```diff
@@ -8,7 +8,7 @@ cache:
     - node_modules
 
 addons:
-  firefox: "46.0"
+  firefox: "47.0"
```

Comment:
I was hitting `yield expressions must be parenthesized` quite often. This was fixed in [Bug 1250589](https://bugzilla.mozilla.org/show_bug.cgi?id=1250589) that landed in 47.
I changed to make testing this issue easier, but I ended leaving it there.